handler/src/main/java/io/netty/handler/ssl/OpenSsl.java

```diff
@@ -404,14 +402,14 @@ static long memoryAddress(ByteBuf buf) {
     private OpenSsl() { }
 
     private static void loadTcNative() throws Exception {
-        String os = normalizeOs(SystemPropertyUtil.get("os.name", ""));
-        String arch = normalizeArch(SystemPropertyUtil.get("os.arch", ""));
+        String os = PlatformDependent.normalizedOs();
+        String arch = PlatformDependent.normalizedArch();
 
         Set<String> libNames = new LinkedHashSet<String>(4);
         // First, try loading the platform-specific library. Platform-specific
         // libraries will be available if using a tcnative uber jar.
         libNames.add("netty_tcnative_" + os + '_' + arch);
-        if (LINUX.equalsIgnoreCase(os)) {
+        if ("linux".equalsIgnoreCase(os)) {
```

Comment:
Should this be replaced with calls to PlatformDependent to get if linux?

Reply:
maybe... I did want to keep the change focused ;)